package.json

```diff
@@ -64,6 +65,7 @@
     "err-code": "^1.1.2",
     "interface-datastore": "~0.7.0",
     "ipfs-block": "~0.8.1",
+    "ipfs-repo-migrations": "github:ipfs/js-ipfs-repo-migrations#master",
```

Comment:
Hmmmmmmmmmm, I think that with the automatic downgrading we will have to wait to release this with some proper migration (eq. the migration-8 which is in the process). Because now `js-ipfs-repo-migration` reports the latest migration it has as `0` because there are no migrations. Now in this auto-downgrade behavior, it will try to downgrade to `0` which will fail........ 🙈

Reply:
Add an empty v7 migration?